recipe-finder-fe/src/Utilities/TextInput/index.tsx

```diff
@@ -0,0 +1,31 @@
+interface TextInputProps {
+  name: string;
+  id: string;
+  placeholder: string;
+  title: string;
+  required?: boolean;
+}
+
+export default function TextInput({
+  name,
+  id,
+  placeholder,
+  title,
+  required
+}: TextInputProps) {
+  return (
+    <>
+      <label htmlFor={id}>
+        {title}
+        <input
+          type="text"
+          name={name}
+          id={id}
+          placeholder={placeholder}
```

Comment:
What if you want a blank placeholder? Could you make it optional and give it a default value? Remember `??` (nullish coalesing operator)

Reply:
Think we have understood this and fixed